### ISSUE TITLE
download from hotstar

### ISSUE THREAD

**Issue by raihanirfan** [KurtBestor/Hitomi-Downloader#4502]
ada yang bisa jelasin cara download dari website ini? tolong ya, terima kasih

**Comment by samleong123** [comment]
I don't think it will work as Hotstar resources are encrypted by DRM by default. I don't think this tool can decrypt DRM as it was illegal to do that.

**Comment by raihanirfan** [comment]
but in support site, you can find hotstar there
http://ytdl-org.github.io/youtube-dl/supportedsites.html
